src/PlaneWaveBasis.jl

```diff
@@ -224,13 +228,13 @@ end
     kweights = T.(model.n_spin_components .* kweights) ./ tot_weight
     @assert mpi_sum(sum(kweights), comm_kpts) ≈ model.n_spin_components
 
-    # Create dummy terms array for basis to handle
-    terms = Vector{Any}(undef, length(model.term_types))
-
+    # Volume element for integration
     dvol = model.unit_cell_volume ./ prod(fft_size)
```

Comment:
isn't this already documented as a field in pwbasis?

Reply:
I don't get what you mean by this comment, sorry. It's the place this quantity is computed.

Reply:
I mean the comment is unnecessary

Reply:
Ah yes ok that might be.